### ISSUE TITLE
Can't play large single file audio book

### ISSUE THREAD

**Issue by darksidemilk** [PaulWoitaschek/Voice#1094]
I have swang song ripped from 40 discs to a single mp3.
It's a little over 2GB and just shy of 40 hours. It shows up in voice and shows the right length. But when I hit play it flashes the button. Like it shows the pause icon for half a second then goes back to the play icon. I can play the file in vlc media player, but that would be a pain to use for listening to an audio book. I'm sure I could split it smaller, but I don't see why the app wouldn't be able to play the file.

**Comment by darksidemilk** [comment]
I found this was a problem listed in some other issues. I was able to fix it by converting back to wav and also worked fine as an aac. More research says that mp3 is old and was superseded by aac (advanced audio codec). Although I realize from other issues that it isn't always easily recreated, I think it's worth noting somewhere in the documentation that it's recommended to use a newer format than mp3 for files 2 GB and larger. I split my mp3 to just under 2 GB and it worked, so there's something to it.
